src/react-components/input/Slider.js

```diff
@@ -28,105 +24,56 @@ function updateGradient(el, min, max, value) {
   }
 }
 
-export function Slider({ min, max, step, digits, value, defaultValue, onChange, disabled, className }) {
-  const myRoot = useRef();
-  const inputRef = useRef();
-  const [displayValue, setDisplayValue] = useState(-1);
-  const [isDragging, setIsDragging] = useState(false);
+export const Slider = memo(
+  forwardRef(({ min, max, step, defaultValue, onChange, disabled, className, ...rest }, ref) => {
```

Comment:
Never really used forwardRef, seems like its being used correctly here but I am still not sure this component actually needs any state. I would expect it just gets passed in a `value` (and not a `defaultValue`) and then the thing using the component is responsolbe for passing in an updated `value` in response to `onChange`